docs/products/postgresql/howto/postgresql-debezium-cdc.rst

```diff
@@ -0,0 +1,192 @@
+Handling PostgreSQL® node replacements when using Debezium for Change Data Capture
+=================================================================================
+
+As an Aiven customer you may be running a Aiven for Apache Kafka® Connect
+cluster that runs one or more `Debezium PostgreSQL® source
+connectors <https://debezium.io/documentation/reference/connectors/postgresql.html>`__
+(Debezium) capturing changes from an Aiven for PostgreSQL® (PG) service.
+
+After the PG service has had a maintenance update or undergo any
+operation which replaces the nodes (such as a plan or cloud region
+change), Debezium would lose connection to the PG database. It is able
+to recover from connection errors to the database and start listening
+for change events again after a restart of the connector's task.
+However, in some cases the Debezium replication slot is lagging behind
+as the amount of WAL logs grows causing a concern.
+
+There is a GitHub repository where you can easily spin up a Debezium -
+PostgreSQL setup to test the node replacement scenario and also follow
+along this help article:
+https://github.com/aiven/debezium-pg-kafka-connect-test
+
+The possible errors encountered, the cause and solution to the growing
+replication slot lag, and how to ensure a graceful PG node replacement
+or maintenance upgrade are explained in the next sections.
+
+.. _h_ab9933dea8:
+
+Possible errors seen in Debezium connector logs when PG has a node replacement
+------------------------------------------------------------------------------
+
+During the reported PG node replacements where Debezium could not
+recover, we have observed the following errors that have occurred in one
+or more occasions:
+
+-  ``org.apache.kafka.connect.errors.ConnectException: Could not create PG connection``
+
+-  ``io.debezium.DebeziumException: Could not execute heartbeat action (Error: 57P01)``
+
+-  ``org.PostgreSQL.util.PSQLException: ERROR: replication slot "SLOT_NAME" is active for PID xxxx``
+
+These are all unrecoverable errors, meaning that they require a restart
+of the connector task(s) in order for the connector to resume operations
+again. A restart can be performed manually either through the web
+console or via the `Kafka Connect REST
+API <https://docs.confluent.io/platform/current/connect/references/restapi.html#rest-api-task-restart>`__
+(you can get the service URI from the web console).
+
+.. image:: pg-debezium-cdc_image1.png
```

Comment:
This link is broken reference must include full path `/images/products/postgresql/pg-debezium-cdc_image1.png`